board/models.py

```diff
@@ -0,0 +1,55 @@
+from django.db import models
+from order.models import *
+from product.models import *
+from user.models import *
+
+class Board(models.Model):
+    uploader        = models.CharField(max_length=45)
+    created_at      = models.DateTimeField(
+                        auto_now = True,
+                        )
+    content         = models.CharField(max_length=45)
+    theme           = models.CharField(max_length=45)
+    class Meta:
+        db_table = "boards"
+
+class BoardImage(models.Model):
+    board       = models.ForeignKey(
+                    'Board',
+                    on_delete = models.CASCADE
+                    )
+    image_url   = models.CharField(max_length=2000)
```

Comment:
```suggestion
    image_url   = models.URLField(max_length=2000)
```
장고에는 URLField라는것이 존재합니다. 
기능으로서 선언을 부탁드리는건 아니고 기왕이면 가독성에 도움이 되는 필드 선언이어서 권장드리고 있습니다.
[장고 모델 필드 레퍼런스](https://docs.djangoproject.com/en/3.1/ref/models/fields/#urlfield) 를 참고해주세요. 다른 테이블의 동일 요소도 전부 변경해주세요.